.github/hex-packages.json

```diff
@@ -8,6 +8,15 @@
     "language": "erlang",
     "authorizedUsers": ["bryannaegele","tsloughter"]
   },
+  "broadway": {
+    "workingDirectory": "instrumentation/opentelemetry_broadway",
+    "name": "Broadway Instrumentation",
+    "packageName": "opentelemetry_broadway",
+    "tagPrefix": "opentelemetry-broadway-v",
+    "buildTool": "mix",
+    "language": "elixir",
+    "authorizedUsers": ["bryannaegele","tomtaylor","tsloughter","whatyouhide"]
```

Comment:
I took the liberty to shove myself in there too 😉

Reply:
:+1: